File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BALANCA/configurar_modelo_balanca.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BALANCA/balanca_command.dart';

class ConfigurarModeloBalanca extends BalancaCommand {
  final int _modeloBalanca;

  ConfigurarModeloBalanca(this._modeloBalanca)
      : super('ConfigurarModeloBalanca');

  @override
  String functionParameters() {
    return '"modeloBalanca"' ':' '$_modeloBalanca';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BALANCA/Commands/ConfigurarModeloBalanca.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BALANCA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.BALANCA.BalancaCommand

class ConfigurarModeloBalanca(val modeloBalanca: Int) : BalancaCommand<Int>("ConfigurarModeloBalanca") {
    override fun functionParameters(): String {
        return "\"modeloBalanca\"" + ":" + modeloBalanca
    }
}